app/src/main/java/com/github/geohunt/app/ui/components/activehunts/ActiveHunts.kt

```diff
@@ -0,0 +1,137 @@
+}
+
+/**
+ * Utility function to show the hunts of a user which is currently getting fetched from the database
+ * @param user the LazyRef instance of the user
+ */
+@Composable
+fun ActiveHunts(user: LazyRef<User>) {
+    Box(modifier = Modifier.fillMaxSize()) {
+        FetchComponent(lazyRef = { user }, modifier = Modifier.align(Alignment.Center)) {resolvedUser ->
+            ActiveHunts(challenges = resolvedUser.challenges)
+        }
+    }
+}
+
+/**
+ * A screen that shows all the active hunts of a user
+ * The hunts are displayed on a horizontal scrollable list
+ * @param challenges the challenges the screen has to display
+ */
+@Composable
+fun ActiveHunts(challenges: List<LazyRef<Challenge>>) {
+    Column(modifier = Modifier
+            .fillMaxSize()
+            .padding(10.dp)){
+        TitleText()
+
+        Spacer(modifier = Modifier.size(10.dp))
+
+        ActiveHuntsList(challenges = challenges)
+    }
+}
+
+/**
+ * The title of the screen, displays "Active Hunts" using the correct colors
+ */
+@Composable
+fun TitleText() {
+    Text(
+            buildAnnotatedString {
+                withStyle(style = SpanStyle(fontSize = 40.sp, fontFamily = Lobster)) {
+
+                    withStyle(style = SpanStyle(color = Color.Black)) {
+                        append("Active")
+                    }
+
+                    append(" ")
+
+                    withStyle(style = SpanStyle(color = geoHuntRed)) {
+                        append("hunts")
+                    }
+                }
+            }
+    )
+}
+
+/**
+ * The list of active hunts
+ * Creates a scrollable list of challenges using the given list
+ * If the list is empty, shows EmptyChallengeScreen
+ * @param challenges the challenges to display
+ */
+@Composable
+fun ActiveHuntsList(challenges: List<LazyRef<Challenge>>) {
+    //wrapper Box
+    Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
+        if(challenges.isEmpty()) {
+            EmptyChallengesScreen()
+        }
+        else {
+            LazyRow(modifier = Modifier.testTag("challenge_row"),
+                    contentPadding = PaddingValues(30.dp, 0.dp),
+                    horizontalArrangement = Arrangement.spacedBy(10.dp)) {
+                items(challenges) { challenge ->
+                    Box(modifier = Modifier.size(300.dp, 600.dp)) {
+                        ChallengePreview(challenge = challenge)
+                    }
+                }
+            }
+        }
+
+    }
+}
+
+/**
+ * The screen to show when they are no challenges to display
+ * Simply displays a text saying no challenges have been selected yet
+ * and a button redirecting to the screen to select new challenges
+ */
+@Composable
+fun EmptyChallengesScreen() {
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        Text(text = "No challenges yet...\n" +
+                "Go pick some challenges to begin your hunt!")
```

Comment:
This is just a note, but for next time, if you could use strings resources in order to ease i18n future rework.